packages/ecommerce-app-base/docs/README.md

```diff
@@ -50,7 +50,7 @@ ___
 
 Function that should return true when the button should be disabled.
 
-**`param`** Currently selected assets
+**`param`** Currently selected skus
```

Comment:
Super nit: Maybe explain the abbreviation here? so we say stock-keeping unit (sku) or something similiar?

Reply:
I've added a sentence to both readmes to explain what the app actually does. Also mentioned stock keeping unit there